cloud/auth.go

```diff
@@ -388,7 +388,7 @@ func (c *client) saveToken() error {
 	return nil
 }
 
-func (c *client) loginWithPassowrd(ctx context.Context) error {
+func (c *client) loginWithPassword(ctx context.Context) error {
```

Comment:
Sometimes projects try to be cute, especially when writing passw0rds, but this didn't seem like such a case...